docs/readthedocs/source/doc/UserGuide/python.md

```diff
@@ -84,9 +84,14 @@ You may test if the installation is successful using the interactive Python shel
 * Try to run the example code below to verify the installation:
 
   ```python
-  from bigdl.orca import init_orca_context
+from bigdl.dllib.nncontext import init_nncontext
```

Comment:
The layout is broken: https://github.com/intel-analytics/BigDL/blob/705c1b24deec64651580fba22cd7a18c952aa2f2/docs/readthedocs/source/doc/UserGuide/python.md

Reply:
fixed